docs/getting-started/cad-software.md

```diff
@@ -47,26 +36,18 @@ If you do not see fields to enter your information on the linked page, try a dif
 
 ![](../../.gitbook/assets/altium4.PNG)
 
-{% hint style="warning" %}
 If the connection is not successful, try the following:
 
 If you are on campus, ensure that you are connected to **UBCSecure** or **UBCPrivate**. 
 
 If you are using the license remotely, ensure that you are connected to myVPN. 
```

Comment:
I believe there is a UBC specific vpn service through cisco, can be downloaded through the UBC software downloads website.

Reply:
myVPN is the UBC vpn through Cisco :)